templates/graphic/app/styles/main.scss

```diff
@@ -1,21 +1,36 @@
-// Variables and mixins
+// base (no output)
+@import '@texastribune/queso-ui/assets/scss/1-settings/all';
+@import '@texastribune/queso-ui/assets/scss/2-tools/all';
 @import 'variables';
-@import 'mixins';
 @import 'functions';
+@import 'mixins';
+
+// resets and root elements
+@import '@texastribune/queso-ui/assets/scss/3-resets/all';
+@import '@texastribune/queso-ui/assets/scss/4-elements/all';
+
+// typography
+@import '@texastribune/queso-ui/assets/scss/5-typography/all';
+@import 'typography';
 
-// Third-party libraries
+// third-party libraries
```

Comment:
Fortunately, we can now get rid of the sass-mq import here (lines 16 and 17). We import it through line 2, which links back to [this file](https://github.com/texastribune/queso-ui/blob/main/assets/scss/2-tools/_all.scss)